core/src/main/java/org/apache/struts2/interceptor/FetchMetadataInterceptor.java

```diff
@@ -0,0 +1,96 @@
+package org.apache.struts2.interceptor;
+
+import com.opensymphony.xwork2.ActionContext;
+import com.opensymphony.xwork2.ActionInvocation;
+import com.opensymphony.xwork2.interceptor.AbstractInterceptor;
+import com.opensymphony.xwork2.interceptor.PreResultListener;
+import com.opensymphony.xwork2.util.TextParseUtil;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * <!-- START SNIPPET: description -->
```

Comment:
It's  not needed anymore, we don't use Confluence any longer, instead you can create a PR with description about the new interceptor [here](https://github.com/apache/struts-site/blob/master/source/core-developers/interceptors.md)